client/ayon_deadline/addon.py

```diff
@@ -38,6 +39,11 @@ def initialize(self, studio_settings):
 
         self.deadline_servers_info = deadline_servers_info
 
+        self._pools_by_server_name = {}
+        self._limit_groups_by_server_name = {}
+        self._groups_by_server_name = {}
+        self.__machines_by_server_name = {}
```

Comment:
Dunder attribute name.
```suggestion
        self._machines_by_server_name = {}
```